packages/wasm/crate/src/metadata.rs

```diff
@@ -0,0 +1,42 @@
+use penumbra_proto::core::asset::v1::Metadata;
+use regex::Regex;
+
+pub static UNBONDING_TOKEN_REGEX: &str = "^uunbonding_(?P<data>start_at_(?P<start>[0-9]+)_(?P<validator>penumbravalid1(?P<id>[a-zA-HJ-NP-Z0-9]+)))$";
+pub static DELEGATION_TOKEN_REGEX: &str =
+    "^udelegation_(?P<data>penumbravalid1[a-zA-HJ-NP-Z0-9]+)$";
```

Comment:
This is a copy of core's [delegation token regex](https://github.com/penumbra-zone/penumbra/blob/de135e8/crates/core/component/stake/src/delegation_token.rs#L67) and [unbonding token regex](https://github.com/penumbra-zone/penumbra/blob/de135e8/crates/core/component/stake/src/unbonding_token.rs#L71), although the unbonding token regex has been modified from the original to be able to extract the ID.

I'm not crazy about having duplicate Regex, so if this PR is merged, I'll make a PR to core to extract the Regex for those tokens so they can be imported here. (I'll also have to modify the unbonding token regex to include the ID.)